src/client/datascience/jupyter/debuggerVariables.ts

```diff
@@ -290,10 +289,12 @@ export class DebuggerVariables extends DebugLocationTracker
         );
         if (results && results.result) {
             // Results should be the updated variable.
+            const info = JSON.parse(results.result);
```

Comment:
Was this just for debugging purposes? This seems equivalent to what we had before.

Reply:
Yeah just for debugging purposes, reverting